terraform/secrets/README.md

```diff
@@ -0,0 +1,17 @@
+### Example dev.tfvars file:
+
+Mandatory variables:
+```
+aws_account_id        = "<ACCOUNT_ID>"
+aws_access_key_id     = "<ACCESS_KEY_ID>"
+aws_secret_access_key = "<SECRET_ACCESS_KEY>"
+domain                = "sdp-dev"
+```
+
+Optional variables that can be set later:
```

Comment:
Can we just include these as mandatory? They need to be set for the app to work no?